backend/apps/accounts/views.py

```
@@ -860,59 +904,62 @@ def get(self, request):
             # Save to buffer
             buffer = io.BytesIO()
-            img.save(buffer, format='JPEG', quality=85)
-            
+            img.save(buffer, format="JPEG", quality=85)
+
             # Create new file
             filename = f"{request.user.id}_avatar.jpg"
             avatar_file = ContentFile(buffer.getvalue(), name=filename)
-            
+
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)